packages/grafana-sql/src/components/NumberInput.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Input } from '@grafana/ui/src/components/Input/Input';
+
+type NumberInputProps = {
+  value: number;
+  defaultValue: number;
+  onChange: (value: number) => void;
+  width: number;
+  placeholder?: string;
+  disabled?: boolean;
```

Comment:
i think `disabled` can be removed because it is never `true`. (when it would become true in `ConnectionLimits.tsx`, it will not be rendered at all... but please doublecheck this 😄 )

Reply:
It seems to be used in the Max idle field when the Auto Max Idle is enabled: 
![image](https://github.com/grafana/grafana/assets/561365/c76c121d-f03b-402a-a243-8e48d8f8f815)

Bit unrelated, should the "Auto Max Idle" be "Auto max idle" matching other fields in this section?

Reply:
hi @jarben , i understand it visually looks like a disabled input field, but in fact that is an `<InlineLabel>`.
if you look at this piece of code:
```ts
        {autoIdle ? (
          <InlineLabel width={labelWidth}>{options.jsonData.maxIdleConns}</InlineLabel>
        ) : (
          <NumberInput
            value={jsonData.maxIdleConns}
            defaultValue={config.sqlConnectionLimits.maxIdleConns}
            onChange={(value) => {
              onJSONDataNumberChanged('maxIdleConns')(value);
            }}
            width={labelWidth}
            disabled={autoIdle}
          />
        )}
```

so, if `autoidle` is:
- `true`, we render am `<InlineLabel>`, we do not render the `<NumberInput>` at all.
- `false`, we render the `<NumberInput>`, with `disabled=false`

based on this, to me it seems,  `<NumberInput>` is always called with `disabled=false` (well, or `disabled=undefined`). so it seems we don't have to handle `disabled=true` at all.
what do you think?

Reply:
and related to changing `Auto Max idle` to `Auto max idle`, i agree 👍 .

Reply:
oh, you are right, overlooked. Updated now, thanks!